conf/modules.config

```diff
@@ -108,7 +108,7 @@
             ext.prefix = { "${meta.id}" }
         }
 
-        withName: "CLUSTALO_ALIGN|FAMSA_ALIGN|LEARNMSA_ALIGN|MAFFT|MAGUS_ALIGN|MUSCLE5_SUPER5|REGRESSIVE|TCOFFEE_ALIGN|TCOFFEE3D_ALIGN|FOLDMASON_EASYMSA"{
+        withName: "CLUSTALO_ALIGN|FAMSA_ALIGN|KALIGN_ALIGN|LEARNMSA_ALIGN|MAFFT|MAGUS_ALIGN|MUSCLE5_SUPER5|REGRESSIVE|TCOFFEE_ALIGN|TCOFFEE3D_ALIGN|FOLDMASON_EASYMSA"{
```

Comment:
Why is foldmason out of alphabetical order?

Reply:
fixed